apps/odyssey-stats/README.md

```diff
@@ -32,7 +32,7 @@ $("#wpcom").on('click', 'a', function (e) {
 The `Gridicon` in `@automattic/components` leverages `<use>` to load SVG sprites and has issues when loading from CDN (i.e. other than the main domain). So we had to replace with one that doesn't load the SVG sprite file - `packages/components/src/gridicon/no-asset.tsx` - and then in Jetpack, we load it separately:
 
 ```
-$.get("https://widgets.wp.com/calypso-happychat/images/gridicons-506499ddac13811fee8e.svg", function(data) {
+$.get("https://widgets.wp.com/images/gridicons-506499ddac13811fee8e.svg", function(data) {
```

Comment:
This is 404-ing. Maybe we can just revert it? 

Also, noting that there are some happychat assets in `widgets.wp.com` that we might want to delete.

Reply:
Ah, I thought the original link was 404-ing as well, but turns out it's a more complex SVG than I expected.
@kangzj, looks like you've added this in https://github.com/Automattic/wp-calypso/pull/69231. Feels like a brittle approach. Thankfully, it seems you've copied them already to your own folder on `widgets.wp.com`, so I've updated the link the README.md to it in https://github.com/Automattic/wp-calypso/pull/78433/commits/db92ddcabfa547dfdbdc98e86866c56a4461365c 🙇